{"nbformat":4,"nbformat_minor":0,"metadata":{"colab":{"name":"Untitled0.ipynb","provenance":[],"authorship_tag":"ABX9TyPaWRcFS1NNRmx10PbTN459"},"kernelspec":{"name":"python3","display_name":"Python 3"}},"cells":[{"cell_type":"code","metadata":{"id":"zWE8HTr1W5KQ","colab_type":"code","colab":{"base_uri":"https://localhost:8080/","height":34},"executionInfo":{"status":"ok","timestamp":1598589549476,"user_tz":-480,"elapsed":792,"user":{"displayName":"William Wong","photoUrl":"https://lh3.googleusercontent.com/a-/AOh14GibA4gcmJFQMv9OnmHKk345FSeiT633JRgfNnpPe8o=s64","userId":"09820637723864922433"}},"outputId":"5f7f7ca6-de8b-4453-a94f-75383212ec05"},"source":["import glob\n","\n","# Importing drive method from colab for accessing google drive\n","from google.colab import drive\n","drive.mount('/content/drive')"],"execution_count":3,"outputs":[{"output_type":"stream","text":["Drive already mounted at /content/drive; to attempt to forcibly remount, call drive.mount(\"/content/drive\", force_remount=True).\n"],"name":"stdout"}]},{"cell_type":"code","metadata":{"id":"-ciE5jOMXP3N","colab_type":"code","colab":{"base_uri":"https://localhost:8080/","height":34},"executionInfo":{"status":"ok","timestamp":1598589594645,"user_tz":-480,"elapsed":1502,"user":{"displayName":"William Wong","photoUrl":"https://lh3.googleusercontent.com/a-/AOh14GibA4gcmJFQMv9OnmHKk345FSeiT633JRgfNnpPe8o=s64","userId":"09820637723864922433"}},"outputId":"16030585-b478-44ac-e03a-1d0aa0b227c2"},"source":["import sys\n","!{sys.executable} -m pip install -r requirements.txt"],"execution_count":6,"outputs":[{"output_type":"stream","text":["\u001b[31mERROR: Could not open requirements file: [Errno 2] No such file or directory: 'requirements.txt'\u001b[0m\n"],"name":"stdout"}]},{"cell_type":"code","metadata":{"id":"g1Z0bmbGXXsX","colab_type":"code","colab":{}},"source":[""],"execution_count":null,"outputs":[]}]}